src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -174,6 +174,11 @@ export default function InstallModal({ appName, backdropClick }: Props) {
     <span className="modalContainer">
       {gameInfo?.game?.title ? (
         <div className="modal">
+          <FontAwesomeIcon
```

Comment:
Please, use an SvgButton component with the FontAwesomeIcon nested. FontAwesomeIcon is just an `<svg>` tag that can't be focused with a keyboard

Here's an example https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/src/screens/Library/components/GameCard/index.tsx#L148-L150

Also an `aria-label` attribute with the text `Close dialog` is a good idea

Reply:
![image](https://user-images.githubusercontent.com/80984726/147787304-3ae7678a-89de-47e0-b731-d227cdfdde6c.png)
Got it, this works for me, what do you think?